docs_guide/sphinx_guide/cross-reference.rst

```diff
@@ -1,12 +1,14 @@
-================================================
-Crossreference to other pages from documentation
-================================================
+=================================================
+Cross-reference to other pages from documentation
```

Comment:
Should this title should be changed to "How to cross-reference to other pages from documentation" like you did for other how-tos?